src/replication.c

```diff
@@ -2617,6 +2676,7 @@ initialize_volume(spec_t *spec, int replication_factor, int consistency_factor)
 	TAILQ_INIT(&spec->rcommon_waitq);
 	TAILQ_INIT(&spec->rq);
 	TAILQ_INIT(&spec->rwaitq);
+	TAILQ_INIT(&spec->identified_replica);
```

Comment:
should we destroy the list during destroy_volume?

Reply:
PR which contains destroy_volume function is not yet merged. I thought of handling destroying identified_replica in that PR #54 once this PR gets merged.